### DISCUSSION TITLE
My Badges don't update

### DISCUSSION THREAD

**Discussion by PedroTsI** [community/community#55075] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

My Badges don't update with the amount achievement do it.

**Comment by NImaism** [comment]
Hello sir, you have **several** options
**1.** Refresh the page: Sometimes badges take a little time to update. Try refreshing the page to see if the badges update.
**2.** Check the badge URL: Make sure the URL you're using to display the badge is accurate. If the URL is incorrect, the badge won't update.
**3.** Check the badge status: Go to the badge URL directly and check if the badge status is correct. If the status is incorrect, then the issue is most likely on the part of the badge provider.
**4.** Clear your browser cache: Clearing your browser cache can sometimes help fix issues with badges not updating.
**I hope your problems will be solved**

**Reply by d-coder111** [reply]
Hey I am also facing the same issue.

**Comment by judahpaul16** [comment]
Try this: https://clear-badge-cache.com/

**Comment by ghost** [comment]
Add cache-control=no-cache as a parameter to your URL. per example:
Old URL = https://tryhackme-badges.s3.amazonaws.com/secutest.png?style=flat
New URL = https://tryhackme-badges.s3.amazonaws.com/secutest.png?style=flat&cache-control=no-cache
